.pipelines/cni/cilium/cilium-scale-test.yaml

```diff
@@ -0,0 +1,367 @@
+pr: none
+trigger: none
+
+stages:
+  - stage: update_daemonset_versions
+    displayName: "Update Cilium + CNS Version and Restart Nodes"
+    jobs:
+      - job: update_version
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                az aks get-credentials --resource-group ${CLUSTER} --name ${CLUSTER}
+                echo "Redeploy all cilium components and update cilium version. Redeploy all to catch all changes between versions"
+                echo "deploy Cilium ConfigMap"
+                kubectl apply -f cilium/configmap.yaml
+                kubectl apply -f test/integration/manifests/cilium/cilium-config.yaml
+                echo "install Cilium ${CILIUM_VERSION_TAG}"
+                envsubst '${CILIUM_VERSION_TAG},${CILIUM_IMAGE_REGISTRY}' < test/integration/manifests/cilium/daemonset.yaml | kubectl apply -f -
+                envsubst '${CILIUM_VERSION_TAG},${CILIUM_IMAGE_REGISTRY}' < test/integration/manifests/cilium/deployment.yaml | kubectl apply -f -
+                kubectl apply -f test/integration/manifests/cilium/cilium-agent
+                kubectl apply -f test/integration/manifests/cilium/cilium-operator
+                echo "Keep CNS version up to date, grabbing pipeline parameter"
+                CNS_IMAGE=${CNS_IMAGE}
+                sed -i '/containers:/{n;n;s/\(image\).*/\1: '"${CNS_IMAGE//\//\\/}"'/}' test/integration/manifests/cns/daemonset.yaml 
+                kubectl apply -f test/integration/manifests/cns/daemonset.yaml 
+                vmss_name=$(az vmss list -g MC_${CLUSTER}_${CLUSTER}_$(LOCATION) --query "[].name" -o tsv)
+                make -C ./hack/aks restart-vmss AZCLI=az CLUSTER=${CLUSTER} REGION=$(LOCATION) VMSS_NAME=$vmss_name
```

Comment:
nit: This would restart the first vmss only. I think we don't have more than one right now, but would be nice to add a loop around the restart vmss. Something like: https://github.com/Azure/azure-container-networking/blob/master/.pipelines/singletenancy/aks/e2e-step-template.yaml#L68-L70